### ISSUE TITLE
Request: Training code release

### ISSUE THREAD

**Issue by ericmday** [ByteDance-Seed/SeedVR#50]
Hi, thanks for releasing SeedVR2 - impressive results!

Are there any plans to release the training code or fine-tuning scripts? Would be great to be able to adapt the model for custom use cases.

Thanks!
